frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -49,8 +51,66 @@ export function PersonModal({ visible, view, onSaveCohort }: Props): JSX.Element
                             justifyContent: 'space-between',
                         }}
                     >
-                        <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-start' }}>
-                            Found {people.count === 99 ? '99+' : people.count} {people.count === 1 ? 'user' : 'users'}
+                        <div
+                            style={{
+                                display: 'flex',
+                                flexDirection: 'column',
+                                width: '100%',
+                                alignItems: 'flex-start',
+                            }}
+                        >
+                            <span style={{ paddingBottom: 12 }}>
+                                Showing{' '}
+                                <b>
+                                    {people.count > 99 ? '99' : people.count} of {people.count}
+                                </b>{' '}
+                                persons
+                            </span>
+                            {featureFlags[FEATURE_FLAGS.PERSONS_MODAL_FILTERING] && (
+                                <>
+                                    <Input.Search
+                                        allowClear
+                                        enterButton
+                                        placeholder="search person by email, name, or ID"
+                                        style={{ width: '100%', flexGrow: 1 }}
+                                        onChange={(e) => {
+                                            setSearchTerm(e.target.value)
+                                            if (!e.target.value) {
+                                                setPeople(
+                                                    firstLoadedPeople?.people,
+                                                    firstLoadedPeople?.count,
+                                                    people.action,
+                                                    people.label,
+                                                    people.day,
+                                                    people.breakdown_value,
+                                                    people.next
+                                                )
```

Comment:
This action and all the values you're passing to it come from `trendsLogic`. Is there any way to simplify this?